anomdetect/src/main/java/com/expedia/adaptivealerting/anomdetect/detect/breakout/algo/edmx/EdmxDetector.java

```diff
@@ -59,6 +63,7 @@ public DetectorResult detect(MetricData metricData) {
 
         val warmup = buffer.remainingCapacity() > 0;
         val result = new EdmxDetectorResult().setWarmup(warmup);
+        val trusted = isTrusted();
```

Comment:
Is there a point to creating a local variable here?

Reply:
Not entirely necessary. It is used below in the return result. It can be `.setTrusted(isTrusted())` instead, but I was following the existing pattern.